interface/patient_file/summary/stats.php

```diff
@@ -223,8 +223,12 @@ function load_location( location ) {
                 echo "  <td colspan='$numcols' style='color:red;font-weight:bold;'>&nbsp;&nbsp;" . htmlspecialchars( $row['title'] . $reaction, ENT_NOQUOTES) . "</td>\n";
         }
         else {
+          if ($key == "medical_problem" || $key == "medication") {
```

Comment:
If you have a long name it staggers the date . I chose the date before name because of this and Immunizations already have it that way.